README.md

```diff
@@ -12,6 +12,8 @@
 The JVM SDK enables developers to use Java 8 methods and objects to communicate with the [commercetools platform](http://www.commercetools.com/) (formerly SPHERE.IO) rather than using plain HTTP calls.
 Users gain type-safety, encapsulation, IDE auto completion and an internal domain specific language to discover and formulate valid requests.
 
+Tested with JDKs: Oracle 1.8.0_221, OpenJDK 1.8.0_221 and Amazon Corretto 8.222.10.1.
```

Comment:
Sorry if I sound too pedantic: I would suggest to add a heading for this (like `# JDK compatibility`). This makes it much easier to refererence it. WDYT?